### PR TITLE
CORE-74143: Fix consentPoliciesIdMap field path in xdm

### PR DESCRIPTION
**Pull request by garya8**
Please link to the issue #…

### CODE REVIEW

File: extensions/adobe/experience/profile/profile-all.schema.json [hunk 1]
```diff
@@ -89,6 +89,26 @@
           }
         },
         "https://ns.adobe.com/experience/profile/consentPoliciesIdMap": {
+          "title": "consent policies map",
+          "meta:status": "deprecated",
+          "type": "object",
+          "meta:xdmType": "map",
+          "additionalProperties": {
+            "title": "consent policy id map",
+            "description": "map of validated fields for a consent policy",
```
Comment (RussStringham) [inline extensions/adobe/experience/profile/profile-all.schema.json:98]: I would change the title and description to something like:
    "title": "Obsolete consent policy id map",
    "description": "this is not the map you should use. see 'ns.adobe.com/consentPoliciesIdMap"

File: extensions/adobe/experience/profile/profile-all.schema.json [hunk 2]
```diff
@@ -89,6 +89,26 @@
...
+            "type": "object",
+            "meta:xdmType": "map",
+            "additionalProperties": {
+              "title": "consented identities per namespace",
+              "description": "List of identities for each namespace in each consent policy",
```
Comment (RussStringham) [inline extensions/adobe/experience/profile/profile-all.schema.json:103]: similar updates here.

**Comment by garya8** [comment]
Closing the PR to pass CLA check
will reopen.

**Comment by RussStringham** [comment]
approved

**Review by ankushkd — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Review by mjasrotia — APPROVED** [review]

**Pull request merged**